moveit_ros/planning/plan_execution/src/plan_execution.cpp

```diff
@@ -161,7 +160,8 @@ void plan_execution::PlanExecution::planAndExecute(ExecutableMotionPlan& plan,
 void plan_execution::PlanExecution::planAndExecuteHelper(ExecutableMotionPlan& plan, const Options& opt)
 {
   // perform initial configuration steps & various checks
-  preempt_requested_ = false;
+  bool preempt_requested = false;
+  preempt_.check_and_clear();  // clear any previous preempt_ request
```

Comment:
Previously, `preempt_requested_` was reset here. We should keep this behaviour, shouldn't we?

Reply:
That's a standard signalling issue: Either you respond to signals that might have been sent even before the call started (my proposal to remove the line), or there is a negligible time between calling the method and `stop()` actually triggering a preemption. There's ways around it of course, but I don't see the need for that here at the moment.

I actually prefer to reset the signal, but `planAndExecute` has the same problem with my old patch.
So now I basically added your previous patch to fix that case too.